jep/0000/README.adoc

```diff
@@ -0,0 +1,227 @@
+= JEP-0000: Readonly system configuration
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Readonly system configuration
+
+| Sponsor
+| link:https://github.com/timja[timja]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| 2020-01-20
+
+| BDFL-Delegate
+| link:https://github.com/oleg-nenashev[oleg-nenashev]
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins-ci.org/browse/JENKINS-12548[JENKINS-12548]
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+This JEP introduces a method to allow system 'configuration' to be viewed but not changed.
+
+== Specification
+
+Any methods that check for `Jenkins.ADMINISTER`, `Jenkins.RUN_SCRIPTS`, `PluginManager.CONFIGURE_UPDATECENTER`, and `PluginManager.UPLOAD_PLUGINS` will be evaluated to determine if the use of the new `Jenkins.SYSTEM_READ` permission type is appropriate. 
```

Comment:
@timja As i understand it, this proposal potentially conflicts with the one proposed by [JEP-223](https://github.com/jenkinsci/jep/tree/master/jep/223).  I _think_ that when a user goes to save any management page, we would end up with the desired result (ie. a user with `Jenkins.CONFIGURE` would be able to persist some parts of the config they were shown, and a user with `Jenkins.SYSTEM_READ` would not be able to persist anything), but this user experience may be confusing to most users.

Reply:
I think we can make it work, possibly some or conditions and additional permission checks may be required in the UI.

Reply:
@timja @mikecirioli 
Following this matter, because the two permissions are not directly related in the permission hierarchy, the only solution I can think is implementing a way of checking if the user has at least one of them. But I feel like we are hitting a design limitation here, and this solution might be a "dirty" workaround.

Thoughts? Maybe there is a simpler solution :D

Apologies if this topic has been already covered in another place

Reply:
I'll have to look up the code later on but I think I have seen some places that allow multiple permission with an or condition